### PR TITLE
Fixes #RHIROS-771 - Added gunicorn to ROS

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

**Review by kgaikwad — COMMENTED** [review]
My bad! I have added a comment but forgot to post it :) 
Thank you @patilsuraj767  for addressing it as per discussion.

File: gunicorn.conf.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bind = '0.0.0.0:8000'
+workers = 2
+accesslog = '-'
```
[the remark this answers is not in the document]
Comment (kgaikwad) [reply gunicorn.conf.py:3]: Addressed already!

**Review by saltgen — COMMENTED** [review]
Rest of the changes look good, thanks @patilsuraj767!

**Review by saltgen — APPROVED** [review]

**Comment by patilsuraj767** [comment]
@saltgen Thanks for reviewing PR. Merging it.

**Pull request merged**